src/accordion.ts

```diff
@@ -0,0 +1,104 @@
+import { LitElement, html, css } from "lit";
+import { customElement, state, property } from "lit/decorators.js";
+
+@customElement("aalam-accordion")
+export class AalamAccordion extends LitElement {
+    static override styles = css`
+       
+    `;
+
+    @property()
+    togglesel: string = ".acc-title";
+
+    @property()
+    bodysel: string = ".acc-body";
+
+    @property({ type: Boolean })
+    nocloseall: boolean = false;
+
+    @property({ type: Boolean })
+    nomultiple: boolean = false;
+
+    @property({ type: Boolean })
+    alwaysopen: boolean = false;
+
+    @property({ type: String })
+    activecls: string = "acc-active";
+
+    @state()
+    open_count: number = 0;
+
+    override render() {
+        return html`<slot >
+
+        </slot>`;
+    }
+
```

Comment:
We should be monitoring the attribute changed callback for intermediate changes in the nocloseall and nomultiple. Imaging we have some many items opened in the accordion and the application sets nomultiple attribute, In that case we need to close all the opened blocks except the first one.